runtime/doc/diff.txt

```diff
@@ -1,39 +1,31 @@
-*diff.txt*      For Vim version 7.4.  Last change: 2014 May 20
+*diff.txt*      For Vim version 7.4.  Last change: 2015 Jan 19
 
 
 		  VIM REFERENCE MANUAL    by Bram Moolenaar
 
 
-				*diff* *vimdiff* *gvimdiff* *diff-mode*
-This file describes the |+diff| feature: Showing differences between two,
-three or four versions of the same file.
+							*diff* *diff-mode*
+This file describes |diff-mode|, which shows the differences between two,
+three, or four versions of the same file.
 
 The basics are explained in section |08.7| of the user manual.
 
-1. Starting diff mode		|vimdiff|
+1. Starting diff mode		|starting-diff-mode|
```

Comment:
I didn't do that because I thought it would cause entering `:h diff-mode` to bring you to the top of the index, as opposed to right above it (line 8). What do you think?